app-text/cherrytree/cherrytree-0.38.4.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
I'm sorry but this is obsolete now. Could you update it for the new version? Please also do the complete background check, including checking for dependency and build system changes.

Reply:
What exactly is obsolete? Can you tell me how to do a complete background check including checking for dependency and build system changes?